olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -490,13 +490,15 @@ spec:
                   name: knative-openshift
               spec:
                 serviceAccountName: knative-operator
-                containers:
-                  - name: cli-downloads
+                initContainers:
+                  - name: cli-artifacts
                     image: registry.ci.openshift.org/openshift/knative-v0.22.0:kn-cli-artifacts
                     imagePullPolicy: Always
-                    ports:
-                      - name: http-cli
-                        containerPort: 8080
+                    command: ['sh', '-c', 'cp /usr/share/kn/**/* /cli-artifacts']
```

Comment:
Do we want to mangle with permissions here? So we could make it a-x to avoid being executable on the server (which would work for the Linux binary anyway only, but when you download it, I guess it will end up with 0644 on the downloaders side anyway). But I'm overcautious here probably.

Reply:
Fair enough, done!